### PR TITLE
Get and process AIS and hydrophone data

### PR DESCRIPTION
**Pull request by ralatsdc**
Also plots time intervals of hydrophone and AIS data by vessel status.

### CODE REVIEW

File: DataScience/AisAudioLabeler.py [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+            start_timestamp = s.group(1)
+        else:
+            start_timestamp = s.group(1)
+        entry["start_timestamp"] = int(start_timestamp)
+        entries.append(entry)
+    hmd = pd.DataFrame(entries).sort_values(by=["start_timestamp"], ignore_index=True)
+    return hmd
+
+
+def main():
```
Comment (jbenjoseph) [inline DataScience/AisAudioLabeler.py:182]: This function has a lot going on in it. Should the graphing stuff be split into a different function?

**Review by jbenjoseph — COMMENTED** [review]
If you want to make these changes I leave up to you.

File: DataScience/AisAudioLabeler.py (hunk 1 above)
Comment (ralatsdc) [reply DataScience/AisAudioLabeler.py:182]: Yes, I agree ... will do on the next branch.

File: DataScience/AisAudioLabeler.py [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    )
+    ch.setFormatter(formatter)
+    root_logger.addHandler(ch)
+
+logger = logging.getLogger("AisAudioLabeler")
+logger.setLevel(logging.INFO)
+
+
+def download_buoy_objects(
+    download_path, bucket, prefix=None, force=False, decompress=False
```
[the remark this answers is not in the document]
Comment (ralatsdc) [reply DataScience/AisAudioLabeler.py:32]: I am not a fan of type hints, JJ. We should discuss.

[the remark this answers is not in the document]
Comment (jbenjoseph) [reply DataScience/AisAudioLabeler.py:32]: That's 100% fine. I'm curious on why though!

**Review by jbenjoseph — APPROVED** [review]

**Pull request merged**